src/views/SpaceProposals.vue

```diff
@@ -81,6 +82,14 @@ watch([proposals, web3Account], () => {
   }
   updateLastSeenProposal(web3Account.value);
 });
+
+const { explore } = useApp();
+const proposalsIs = computed(() => {
```

Comment:
what does proposalsIs mean?

Reply:
@midgerate if there are proposals, then their number is returned or undefined, I use them in boolean format.  From there and the name proposals is. I mean - is there a proposal.

if you have any suggestions on how to call computed it more correctly, then I will gladly correct

Reply:
I am still confused by your explanation. From your code, you are returning the storing the list of proposals in the proposalsIs variable right? 

Why not call it just `proposals` ?

Reply:
@midgerate The variable proposals is already used in the code, and computed only returns the number of proposals or undefined, not the proposals themselves, it might be better to call hasProposals.  How do you think?

Reply:
then call it `nbbrProposals` or `proposalsCount` and return 0 if there are no proposals.